src/odemis/model/_metadata.py

```diff
@@ -108,6 +112,14 @@
 MD_POL_RHC = "rhc"  # (string) polarization analyzer position
 MD_POL_LHC = "lhc"  # (string) polarization analyzer position
 
+MD_STREAK_TIMERANGE = "Streak Time Range"  # Time range for one streak/sweep
```

Comment:
I'd suggest to move all these (5) metadata related to the detector settings to the top of the metadata.
MD_TIME_RANGE_TO_DELAY can go near MD_RESOLUTION_SLOPE et al, which are for configuration.